subiquity/server/controllers/ubuntu_pro.py

```diff
@@ -34,20 +34,20 @@
 )
 from subiquity.server.controller import SubiquityController
 
-log = logging.getLogger("subiquity.server.controllers.ubuntu_advantage")
+log = logging.getLogger("subiquity.server.controllers.ubuntu_pro")
 
 TOKEN_DESC = """\
 A valid token starts with a C and is followed by 23 to 29 Base58 characters.
 See https://pkg.go.dev/github.com/btcsuite/btcutil/base58#CheckEncode"""
 
 
-class UbuntuAdvantageController(SubiquityController):
-    """ Represent the server-side Ubuntu Advantage controller. """
+class UbuntuProController(SubiquityController):
+    """ Represent the server-side Ubuntu Pro controller. """
 
-    endpoint = API.ubuntu_advantage
+    endpoint = API.ubuntu_pro
 
-    model_name = "ubuntu_advantage"
-    autoinstall_key = "ubuntu-advantage"
```

Comment:
We have shipped subiquity where ubuntu-advantage was a valid autoinstall key, and fixing autoinstall errors is kind of hard for end users.
I would like to have a light compatibility layer that is willing to read in something under the ubuntu-advantage name and populates the value under ubuntu-pro.

Reply:
Good point, thanks!